module_analysis/__manifest__.py

```diff
@@ -8,11 +8,12 @@
     " to know which installed modules comes from Odoo Core, OCA, or are"
     " custom modules",
     'author': 'GRAP, Odoo Community Association (OCA)',
+    'maintainers': ['legalsylvain'],
     'website': "https://github.com/OCA/server-tools/",
     'version': '12.0.1.0.2',
     'license': 'AGPL-3',
     'depends': [
-        'base',
+        'base_automation',
```

Comment:
Why is the dependency on `base_automation` added? Is what you need from ir.cron provided by `base`?

Reply:
1 year later ... 
You're right ! Sorry for the delay. I tested with ``base`` dependency and it works. 
maybe a residual from another implementation I tried...

reverted.